test/index.server.spec.tsx

```diff
@@ -1,3 +1,7 @@
+/**
+ * @jest-environment jsdom
+ */
```

Comment:
This is admittedly weird, but this directive is needed because `@nteract/mockument` is loaded before every Jest test (and it's doing a few setup things that even the server version of this package seems to dpeend on).

In an ideal world, we'd probably remove this and put a little more thought into what the server-side rendering environment looks like.